client/my-sites/stats/stats-navigation/index.jsx

```diff
@@ -22,16 +21,6 @@ class StatsNavigation extends Component {
 		] )
 	}
 
-	componentDidMount() {
-		const slug = this.props.site ? this.props.site.slug : '';
-		siteStatsStickyTabActions.saveFilterAndSlug( this.props.section, slug );
```

Comment:
Likewise, if we wanted to populate a redux value, this is another place we'd have to dispatch an action to do so. For now, 💥